packages/vite/src/node/plugins/manifest.ts

```diff
@@ -101,10 +101,18 @@ export function manifestPlugin(config: ResolvedConfig): Plugin {
         return manifestChunk
       }
 
+      function createAsset(chunk: OutputAsset): ManifestChunk {
+        return {
+          file: chunk.fileName
```

Comment:
Having `src` here if it exists would be helpful too.

Reply:
Wouldn't be the src same as the name of the asset? Also not sure how to do this, asset returns only `name`

```js
if (chunk.name) {
  manifestAsset.src = chunk.name
}
 ```  

Would this be correct? I don't know how to get full path for assets, they don't include `facadeModuleId`